Makefile

```diff
@@ -70,7 +70,7 @@ endif
 govet:
 ifneq ($(HAS_SERVER),)
 	@echo Running govet
-	@# Workaround because you can't install binaries without adding them to go.mod
+	@# Workaround because you cant install binaries without adding them to go.mod
```

Comment:
nit:
```suggestion
	@# Workaround because you can't install binaries without adding them to go.mod
```

Reply:
:( I was hoping you wouldn't mind this. Can I change it to "cannot"? The apostrophe breaks my editor's syntax highlighting.

Reply:
Haha, yeah that's fine

Reply:
:pray: thank you hahaha